plugins/generator-1.18.2/forge-1.18.2/templates/livingentity/livingentity_renderer.java.ftl

```diff
@@ -131,6 +131,21 @@ public class ${name}Renderer extends <#if humanoid>Humanoid</#if>MobRenderer<${n
 	    }
 	</#if>
 
+	<#if data.mobScale??>
+	@Override protected void scale(${name}Entity _ent, PoseStack poseStack, float p_116462_) {
+		<#if hasProcedure(data.mobScale)>
+			Entity entity = _ent;
+			Level world = entity.level;
+			double x = entity.getX();
+			double y = entity.getY();
+			double z = entity.getZ();
+			poseStack.scale((float) <@procedureOBJToNumberCode data.mobScale/>, (float) <@procedureOBJToNumberCode data.mobScale/>, (float) <@procedureOBJToNumberCode data.mobScale/>);

```

Comment:
You should not call the procedure 3 times separately (as this can result in different values if the procedure uses a counter or random numbers) but instead create a local variable with the returned value and use it for all parameters.